google-cloud-firestore/clirr-ignored-differences.xml

```diff
@@ -314,4 +314,11 @@
     <method>com.google.cloud.firestore.Query getQuery()</method>
   </difference>
 
+    <!-- Relocate constants in TraceUtil -->
+  <difference>
```

Comment:
waiting for https://github.com/googleapis/java-firestore/pull/1869 to be merged to see if "internalApi" will lift the clirr restriction on the contstants removal, and add internalStream parameter change on.